### ISSUE TITLE
double entries in my Sierra dock

### ISSUE THREAD

**Issue by mrjayviper** [kcrawford/dockutil#63]
Please refer to this screenshot below.

The 2 icons on the left (MP4 and red icon) was created using dockutil --add /Applications/my-app.app. When I click on the icon on the MP4 icon, another icon showed up (the one of the right). But when I clicked on the red icon, it didn't create a new one like the MP4. is this a problem with Sierra or the script?

Thank you so much.

<img width="393" alt="screen shot 2016-10-03 at 9 24 09 pm" src="https://cloud.githubusercontent.com/assets/5241882/19035301/3fbf6d1c-89b0-11e6-8f46-7fcc18db047d.png">

**Comment by mrjayviper** [comment]
I forgot to add that if I:

1. move the newer MP4 icon and placed it beside the MP4 icon created dockutil
2. remove the MP4 created by dockutil

clicking the remaining MP4 icon will not result in two icons again.

**Comment by WardsParadox** [comment]
@mrjayviper If you clear all MP4 icons from the dock and drag the app you specified to the dock, does launching it also create a second icon?

**Comment by TechGeekMac** [comment]
@mrjayviper Did you ever get this resolved? 

As I have encountered the same issue with a 10.12 build in my workplace (we have only just started to build with 10.12). 

Had no issue with 10.11, but in 10.12 selected apps (not all of them) are launching with an additional icon. I can't see any permissions/ownerships/groups issue or difference between the ones causing an issue to those that are not. 

Very odd, I'm running out of ideas as to what could be causing the issue! My next stage is to try with an older Dockutil version, but my hunch is that its something to do with 10.12 instead.
